### PR TITLE
Fix grammar and spelling mistakes

### PR DESCRIPTION
**Pull request by alti21**
## Pull request content
- [ ] ENANCHEMENT
- [x] BUGFIX

**Related task:** <put link here>

Describe here thecnical detail omitted on task

## Notes for admin
What the pull request admin must know before review it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,14 +15,14 @@ RawCMS is a headless CMS written in ASP.NET Core, built for developers that embr
 - 👪 [Community Chat](https://gitter.im/arduosoft/RawCMS-Headless-CMS-Aspnet) - Gitter
 
 ## Contribution
-As every opensource project, we are looking for contributors. Any kind of help are welcome. *just a star to the project is a lot* If you want to contributute as developer, you can join the project  by [filling this form](https://forms.gle/dddbHWzcxypN9rpx9) or opening an issues. Any other kind of contirbution, from doc to test are also welcome.
+As every for open source project, we are looking for contributors. Any kind of help are welcome. *just a star to the project is a lot* If you want to contributute as developer, you can join the project  by [filling this form](https://forms.gle/dddbHWzcxypN9rpx9) or by opening an issue. Any other kind of contirbution, from docs to tests, are also welcome.
```
Comment (shayneofficer) [inline README.md]: Any kind of help are welcome. => Any kind of help is welcome.
*just a star to the project is a lot* => *Just starring the project is a lot.*
filling this form => filling out this form
 Any other kind of contirbution =>  Any other kind of contribution
are also welcome. => is also welcome

File: README.md [hunk 2]
```diff
@@ -15,14 +15,14 @@ RawCMS is a headless CMS written in ASP.NET Core, built for developers that embr
...
 
 ## Try it
-You can test RawCMS using the public [demo](http://rawcms-demo.herokuapp.com/) an default credential (username:bob, password:XYZ).
+You can test RawCMS using the public [demo](http://rawcms-demo.herokuapp.com/) using the default credentials (username:bob, password:XYZ).
 
 ## Deploy it
 
-You can deploy it using many options. Rawcms is dockerized from the development stage so modern hosting are fully supported.
+You can deploy it using many options. Rawcms is dockerized from the development stage so modern hosting is fully supported.
```
Comment (shayneofficer) [inline README.md]: Rawcms => RawCMS

File: README.md [hunk 3]
```diff
@@ -15,14 +15,14 @@ RawCMS is a headless CMS written in ASP.NET Core, built for developers that embr
...
 
 1. *Docker Compose* using the provided docke-compose file
```
Comment (shayneofficer) [inline README.md]: docke-compose => docker-compose

**Comment by zeppaman** [comment]
Hi @alti21 and @shayneofficer, thank you so much. This help is a lot for us.

**Comment by zeppaman** [comment]
@shayneofficer I notice only now that your comments are additional fix. It would possible for you to fork this repo, fix it, and then PR?

**Comment by shayneofficer** [comment]
Yeah sure!

**Pull request merged**